test-requirements.txt

```diff
@@ -9,4 +9,6 @@ pep8>=1.7.0
 flake8>=2.2.4
 stestr
 
+jsonschema
```

Comment:
I'd add a short comment here as to why we want to add the test dependency rather than just mocking it out. (I.e. as we discussed we use it to actually validate a schema embedded in the code).